### PR TITLE
feature(dashboard): withdrawal deregistration warning dialog revamp

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

It was necessary to change a default style of the Modal component in order to implement the new style of the withdrawal warning

##### Ticket

YOMO-1532

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/legacy/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -73,7 +80,12 @@ export const Dashboard = () => {
       )
       return
     }
-    setShowWithdrawalDialog(true)
+
+    openModal(
+      strings.warningModalTitle,
+      <WithdrawStakingRewards wallet={wallet} onSuccess={() => resetToTxHistory()} onCancel={() => closeModal()} />,
+      windowHeight - (20 / 100) * windowHeight,
```
Comment (jorbuedo) [inline apps/wallet-mobile/src/legacy/Dashboard/Dashboard.tsx]: I think `windowHeight * 0.8` would be clearer

Comment (banklesss) [reply apps/wallet-mobile/src/legacy/Dashboard/Dashboard.tsx]: yes you are right

`windowHeight - (20 / 100) * windowHeight = windowHeight * (1 -  0.2) = windowHeight * 0.8`

Comment (banklesss) [reply apps/wallet-mobile/src/legacy/Dashboard/Dashboard.tsx]:
```suggestion
      windowHeight * 0.8
```

**Comment by jorbuedo** [comment]
In Android,

- space in the bottom is too much
- there's no horizontal padding
- title of the modal seems out of place on the second step of the process

image

image

**Comment by banklesss** [comment]
@jorbuedo thanks for the check 👍🏻

yes, that's the second dialog. I missed that one

**Comment by banklesss** [comment]
changed to draft to implement the missing dialog

**Pull request merged**